packages/opentelemetry-node/src/spanprocessor.ts

```diff
@@ -10,19 +10,19 @@ import { SENTRY_DYNAMIC_SAMPLING_CONTEXT_KEY, SENTRY_TRACE_PARENT_CONTEXT_KEY }
 import { isSentryRequestSpan } from './utils/isSentryRequest';
 import { mapOtelStatus } from './utils/mapOtelStatus';
 import { parseSpanDescription } from './utils/parseOtelSpanDescription';
-import { clearOtelSpanData, getOtelSpanData } from './utils/spanData';
 
-export const SENTRY_SPAN_PROCESSOR_MAP: Map<SentrySpan['spanId'], SentrySpan> = new Map<
-  SentrySpan['spanId'],
-  SentrySpan
->();
+export const SENTRY_SPAN_PROCESSOR_MAP: Map<string, SentrySpan> = new Map<string, SentrySpan>();
```

Comment:
Actually the type `SentrySpan['spanId']` is not really correct/helpful here, as we are mapping an _otel_ span ID to a sentry Span 🤔